content/en/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -39,6 +39,10 @@ request a particular class. Administrators set the name and other parameters
 of a class when first creating VolumeSnapshotClass objects, and the objects cannot
 be updated once they are created.
 
+{{< note >}}
+Installation of the CRDs are the responsibility of the Kubernetes distribution. Without the required CRDs present the `VolumeSnapshotClass` below will not get created.  
```

Comment:
```suggestion
Installation of the CRDs are the responsibility of the Kubernetes distribution. Without the required CRDs present the `VolumeSnapshotClass` fails. 
```

https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense

This comment is non-blocking, as I could be sold on this applying the rule's exception: "Use future or past tense if it is required to convey the correct meaning."